msgType.h

```diff
@@ -0,0 +1,133 @@
+#pragma once
+typedef enum msgType	//type of messages used as enum
+{
+	joinRequest=0x01,	//used in pairing mode for join request
+	joinAccept,			//used in pairing mode for join accept
+	joinConfirm,		//used in pairing mode for join confirmation
+	configReq,			//used in configuration mode for requesting configuration message
+	configMsg,			//used in configuration mode for sending configuration message
+	configAck,			//used in configuration mode for acknowledging configuration successfull
+	sensorData,			//used in monitoring mode for sending sensor data and battery level
+	ack,				//used for acknowledging a message received
+	motorStatus,		//used for acknowleging status of motor
+	alarm,				//used to send warning messages/error messages
+	initController		//used to notify level monitor that the controller is recently initiated
+}msgType; 
+
+typedef enum devType {
+	monitor = 0x01,
+	controller,
+	relayUnit
+}devType;
+
+typedef enum response {
+	failure = 0x00,
+	success
+}response;
+
+typedef enum motorState {
```

Comment:
Use deviceOn etc for avoiding confusion

Reply:
ok

Reply:
typedef enum motorState {
	DEVICE_ON = 0x00,
	DEVICE_OFF,
	DRY_RUN
}motorState;